test/util/login-helpers.js

```diff
@@ -52,16 +51,29 @@ function initUserMock(level) {
 }
 /* eslint-enable camelcase */
 
-function logIn(client, url, level) {
-  initUserMock(level);
+let tokenCounter = 0;
+
+function getUserToken() {
```

Comment:
That seems fine to me.  (If I've learned anything about Selenium in the last few days, it's to not rely on its APIs...)